### DISCUSSION TITLE
Student Pack refused

### DISCUSSION THREAD

**Discussion by podpah** [community/community#51561] [GitHub Education]
Is there a way to get a human to review my application? I don't have an ID or email due to the nature of my school, but they are listed with Github. And yet when I upload screenshots of their portal with their name, category of my learning and the end date for it, my application gets denied for not having my school name, which it does. It also suggests to take a photo instead as they are less easily manipulated, but I tried that and it also got denied.

Is there a way to get past that bot auto check?

**Comment by samus-aran** [comment]
Hi @podpah  👋

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

**Reply by podpah** [reply]
I got the following:
The image you selected does not appear to contain your school name - it does 
Your document does not appear to include a date demonstrating current academic affiliation - it does

The portal URL has the name of the school in the link, and a logo with the school name. It also states when I started the programme, and when I finish the programme. 
I don't have a school ID or email due to the nature of the programme, and the lack of a need for them, so my only option is the portal which has the evidence required and is also accepted by other places offering student benefits, such as UNiDays
